.github/workflows/integration.yaml

```
@@ -0,0 +1,48 @@
+name: Integration
+on: [push, pull_request]
+env: 
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7, 2.7]
```

Drop 2.7? It's 2020 :wink: 

```suggestion
        python-version: [3.7]
```

---

More than happy to, but I do want to see them green once at least.